src/address.rs

```diff
@@ -119,6 +120,10 @@ impl Error for Disconnected {}
 pub struct Address<A, Rc: RefCounter = Strong> {
     pub(crate) sender: Sender<AddressMessage<A>>,
     pub(crate) ref_counter: Rc,
+    /// Activates once all contexts on this address have been dropped.
+    /// Used in [`Address::attach_stream`](struct.Address.html#method.attach_stream) to stop waiting
+    /// for new messages when this `Address` becomes disconnected and in [`Address::join`].
+    pub(crate) tripwire: Tripwire
```

Comment:
Not sure whether adding this to the address itself is a good idea, since it would inflate the size of the address